lib/generators/factory_bot/authentication/templates/users.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :user do
+    password { "password123" }
+    password_confirmation { "password123" }
+    email_address { "user#{SecureRandom.hex(3)}@example.com" }
+  end
+end
```

Comment:
This file is never used, I made PR https://github.com/thoughtbot/factory_bot_rails/pull/544 to remove it.